src/PhpSpreadsheet/Shared/File.php

```diff
@@ -141,4 +141,25 @@ public static function assertFile($filename)
             throw new InvalidArgumentException('Could not open "' . $filename . '" for reading.');
         }
     }
+
+    /**
+     * Deletes given file if an existing its exists and is NOT BEING USED BY SOME APPLICATIONS.
+     *
+     * @param string $file
+     *
+     * @returns false If file could not be deleted.
+     */
+    public static function deleteFile($file)
```

Comment:
This should be called just "delete" and never return anything, but instead throw exception in case of failure.

Also the message while "not being used by some application" is correct, we should first mention file permissions, which would be a more common case of errors (only Windows prevent a file to be deleted if it's in use, and PhpSpreadsheet would typically not be used on a Windows machine).